lib/routers/places.js

```diff
@@ -21,22 +22,12 @@ const submit = (action) => {
   };
 };
 
-const validateSchema = () => {
-  return (req, res, next) => {
-    const ignoredProps = ['comments'];
-    let data = { ...req.body, establishmentId: req.establishment.id };
-
-    if (res.place) {
-      data = Object.assign({}, res.place, data);
-    }
-
-    data = omit(data, ignoredProps);
-
-    const { Place } = req.models;
-    const error = Place.validate(data);
-
-    return error ? next(error) : next();
-  };
+const validatePlace = (req, res, next) => {
+  return validateSchema(req.models.Place, {
+    ...(res.place || {}),
+    ...omit(req.body, 'comments'),
```

Comment:
This is just a personal preference thing, but I think sometimes it's worth having named vars to add context, even if it's more verbose.

In general I'll always trade a bit of extra code for readability, but I'm coming from a background where file size never really mattered :)